Dockerfile

```diff
@@ -7,8 +7,10 @@ ENV PORT=3000
 
 COPY package.json .
 
-RUN npm install --production
-
+RUN apk add --no-cache --virtual .build-deps make gcc g++ python \
```

Comment:
@dfarache what is this suppose to do?

Reply:
@carocad 
one of the node modules require gcc for compilation. since alpine doesn't include this stuff by default, we're installing it.